.golangci.yml

```diff
@@ -1,27 +1,59 @@
 # Copyright KubeArchive Authors
 # SPDX-License-Identifier: Apache-2.0
 ---
+version: '2'
 run:
-  timeout: 10m
   build-tags:
     - integration
 linters:
   enable:
     - bodyclose
     - containedctx
     - fatcontext
-    - goimports
     - gosec
     - nilerr
     - nilnil
+    - unparam
     - usestdlibvars
     - wastedassign
-    - unparam
-linters-settings:
-  govet:
-    enable-all: false
-    disable-all: false
-    enable:
-      - loopclosure
-      - nilness
-      - shadow
+  settings:
+    staticcheck:
+      checks:
+        # defaut
+        - all
+        - -ST1000
+        - -ST1003
+        - -ST1016
+        - -ST1020
+        - -ST1021
```

Comment:
"The documentation of an exported type should start with type’s name"

If that doesn't enforce having the docs and just keeping consistency among the already in-place docs I'd prefer to keep this.